src/java.base/share/classes/sun/security/provider/certpath/RevocationChecker.java

```diff
@@ -740,9 +746,43 @@ private void checkOCSP(X509Certificate cert,
                         null, -1);
                 }
 
+                List<Extension> tmpExtensions = Collections.<Extension>emptyList();
```

Comment:
One other comment - I think you can just set this to null.

Reply:
Fixed.